pkg/cri/sbserver/sandbox_run.go

```diff
@@ -27,14 +27,15 @@ import (
 	"strings"
 	"time"
 
-	eventtypes "github.com/containerd/containerd/api/events"
-	"github.com/containerd/containerd/protobuf"
-	sb "github.com/containerd/containerd/sandbox"
 	"github.com/containerd/go-cni"
 	"github.com/containerd/typeurl"
 	"github.com/sirupsen/logrus"
 	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
 
+	eventtypes "github.com/containerd/containerd/api/events"
+	"github.com/containerd/containerd/protobuf"
+	sb "github.com/containerd/containerd/sandbox"
```

Comment:
I believe this import alias is unnecessary, or at least unhelpfully terse. The other `sandbox` is aliased to something more informative.

Reply:
The alias is preexisting, the imports were just reorganized. You can see the existing import on line 32 of the old file.